backend/orchestrators/Orchestrator.py

```diff
@@ -7,10 +7,9 @@
 from dotenv import load_dotenv
 
 from backend.conversationtelemetry import ConversationTelemetryClient
+load_dotenv()
```

Comment:
I took this out in a previous PR because it's already in app.py. Could this be removed? I actually had better success without it.

Reply:
Based on our convo, I guess leave it in!